### ISSUE TITLE
[Bug]:  Panic when receive `kill -s SIGINT`

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#4752]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):992765a7eeb960f02c9dc64db5b7339c001ea852
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/77312370/187134954-1b0384f9-7abd-4374-8e6a-316f7c1d7b52.png">

### Steps to Reproduce

```Markdown
1. cd matrixone
2. make config
3. go test -c -tags service_test -cover -covermode=count -coverpkg=./pkg/... ./cmd/mo-service/ -o service_test
4. ./service_test -test.coverprofile=cover.out &>service.log &
5. run bvt
6. ps aux|grep 'service_test' | grep -v 'grep' | awk '{print $2}' | xargs kill -s SIGINT
```

**Comment by zhangxu19830126** [comment]
fixed by #4768

**Comment by zhangxu19830126** [comment]
@sukki37 check
